### ISSUE TITLE
[Lumber] Users want to know how to upgrade their Forest liana version.

### ISSUE THREAD

**Issue by arnaudbesnier** [ForestAdmin/forest-doc#1]
`npm install --save forest-express-sequelize@latest`

**Comment by louisteenbrink** [comment]
@arnaudbesnier you can find this information in the knowledge base section "Update" - https://doc.forestadmin.com/knowledge-base.html#update

**Comment by arnaudbesnier** [comment]
Yes but I think it deserve a small section just below this one: https://doc.forestadmin.com/developer-guide/rails.html#installation
